src/main/java/com/bigludo/ServerHandler.java

```diff
@@ -22,4 +26,19 @@ public void sendMessage(String message, IClient sender) {
             }
         }
     }
+
+    @Override
+    public void run() {
+        boolean stopped = false;
+        try {
+            ServerSocket serverSocket = new ServerSocket(8000);
+
+            do {
+                Socket clientSocket = serverSocket.accept();
+                clientHandlers.add(new ClientHandler("ett-namn", clientSocket, this));
```

Comment:
på metodanropet:

parametrarna är ju (ID, Socket, IServer)
ID ="ett namn" (exempel, förstår den)
Socket = clientSocket (clientens socket, förstår den)
IServer = this (Syftar det till "this server" eller vad antyder "this" en koppling till)

Reply:
This syftar på den instans man exekverar inuti, dvs. vi är ju en instans av en ServerHandler-class.
ServerHandler-klassen implementerar IServer. Om vi skriver this som argument så skickar vi in en "pekare" (ett handtag) till den klassen vi är, dvs. ServerHandler objektet(instansen). Eg. den vi namngett i main, men det är bara ett variabelnamn. Blir lite enklare om man visar hur det ser ut på "minnet" i datorn, att en instans tar upp ett minnesområde, man kan referera till det området på många olika sätt. variabel namn är ett sätt. this är ett annat.

Reply:
Sedan så "ett-namn" är vi inte helt nöjd med, här behöver vi veta vilken klient som ansluter. Om det nu ansluter två klienter så kommer de heta "ett-namn". Vi behöver försöka hitta en dynamik och en metod hur vi kan döpa/namnge en anslutning. Men vi tar det som en "uppgift"(issue).

Reply:
Så på "ett-namn" kan man tänkas behöva en metod som hämtar namnet på anslutningen / genererar ett namn?